### PR TITLE
Automatically fetch Wikipedia dataset and insert it into the databases

### PR DESCRIPTION
**Pull request by avborup**
- [x] Fetch Wikipedia data
- [x] Extract articles from XML
- [x] Insert articles into Postgres
   - [x] Create full text search index
- [x] Insert articles into ElasticSearch

Not entiiiiiirely automatic. Have to comment/uncomment some code parts depending on what to run, and you need to change the code to connect to the correct postgres/elasticsearch instances.

### CODE REVIEW

File: project2/Dockerfile.elastic [hunk 1]
```diff
@@ -5,3 +5,4 @@ EXPOSE 9300
 
 # https://www.elastic.co/guide/en/elasticsearch/reference/7.5/docker.html
 ENV discovery.type=single-node
+ENV xpack.security.enabled=false
```
Comment (arnenator) [inline project2/Dockerfile.elastic:8]: Nice 😎

File: project2/load-data/.gitignore [hunk 2]
```diff
@@ -0,0 +1 @@
+articles/*
```
Comment (arnenator) [inline project2/load-data/.gitignore:1]: Would it be nice to actually have the data in the repo to be completely transparent in the repo and show what data has been used to actually run our experiments?

Or would it be too large?

**Review by arnenator — APPROVED** [review]
I think it is nice! Good job.

**Comment by arnenator** [comment]
I know i have provided a review but since you posted it in Discord I assum that it is ready for review even though it is in draft

File: project2/load-data/.gitignore (hunk 2 above)
Comment (avborup) [reply project2/load-data/.gitignore:1]: It's 28GB decompressed + 7GB compressed, so way too much to store in GitHub 😅

File: project2/Dockerfile.elastic (hunk 1 above)
Comment (avborup) [reply project2/Dockerfile.elastic:8]: ja tak

**Pull request merged**